src/Cache.js

```diff
@@ -62,11 +62,13 @@ class Cache {
    * remove from cache after maxAge time has passed
    *
    * @param {*} key the key to remove
-   * @param {number} maxAge the time in milliseconds to wait before removing key
+   * @param {number} maxAge the time in milliseconds to wait before removing the key
+   * @param {Function} onExpiration a callback that is called after removing the key
    */
-  expireAfter(key: any, maxAge: number) {
+  expireAfter(key: any, maxAge: number, onExpiration: ?Function) {
     setTimeout(() => {
       this.remove(key);
+      if (onExpiration) onExpiration(key.key);
```

Comment:
I'll give `eslint-prettier` a try, I hadn't found that prior and that is more what I was hoping for (combining `prettier` with existing `eslint` style rules). The setup I was recommended by someone else was to apply `eslint-config-prettier`, which basically removed all `eslint` styles and let prettier handle everything ... which is less strict than what I was hoping for.

Don't worry about that for this PR as its outside the scope, but appreciate the recommendation. Once I merged your changes I'll try to reintroduce `prettier` prior to release. Thanks!